### ISSUE TITLE
"crosstab not supported" when using with Browserify

### ISSUE THREAD

**Issue by rikukissa** [tejacques/crosstab#37]
>Uncaught Error: crosstab not supported: localStorage not availabe, addEventListener not available, localStorage.setItem not allowed

The context variable `this` passed to the factory function refers to module.export instead of window object in Browserify environment.

https://github.com/tejacques/crosstab/commit/2bf8403b77b2612680fa44e4014d5b2ff1d41938 seems to cause this so the problem exists only in 0.2.8

**Comment by tejacques** [comment]
Yeah that's definitely a problem.

I think it can be fixed by replacing `this` with:

```.js
(function() { try { return window; } catch(e) { return global; } })()
```

I will make this change today and push it to npm

**Comment by tejacques** [comment]
Commit with fix is in -- will push to npm shortly as 0.2.9
